spec/factories/valid_draft_factories.rb

```diff
@@ -815,7 +819,7 @@
           'Eon' => 'test 3 Eon text 1'
         }]
       }],
-      'ISOTopicCategories' => %w(farming climatologyMeteorologyAtmosphere health)
+      'ISOTopicCategories' => %w[farming climatologyMeteorologyAtmosphere health]
```

Comment:
I think they updated the style guide to use `[]`, https://github.com/bbatsov/ruby-style-guide#percent-w.